Assets/Scripts/Models/Area/World.cs

```diff
@@ -242,14 +242,14 @@ public Tile GetFirstTileWithNoInventoryAround(int maxOffset, int centerX, int ce
             {
                 offsetY = offset;
                 tile = GetTileAt(centerX + offsetX, centerY + offsetY, centerZ);
-                if (tile.Inventory == null)
+                if (tile.Inventory == null && tile.Furniture == null)
```

Comment:
Model in most c# pattern are data repositories and shouldn't modify data from other model, this is why we have controller. But I see your point in the way we use them. Feel free to propose an update ;)